Part3_icon-vis/ex3_0_icon-vis.ipynb

```diff
@@ -0,0 +1,192 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8822930b",
+   "metadata": {},
+   "source": [
+    "# Exercise 3.0: icon-vis\n",
+    "prepared by A. Lauber\n",
+    "\n",
+    "In this exercise, we will work with the GitHub repository [icon-vis](https://github.com/C2SM/icon-vis), which was developed by C2SM and MeteoSwiss. It is a collection of easy-to-use Python scripts for plotting data on unstructured grids like the ICON grid. It is based on the library [psyplot](https://psyplot.github.io), which was already introduced in [Exercise 2.2](../Part2_Mapplots/ex2_2_psyplot.ipynb).\n",
+    "\n",
+    "Advantages of plotting on the unstructured grid:\n",
+    "- Shows the real output without interpolation\n",
+    "- No preprocessing of the data necessary\n",
+    "\n",
+    "Disadvantages of plotting on the unstructured grid:\n",
+    "- It can be quite slow depending on the size of your netCDF file\n",
+    "- Not all features are available yet (psyplot is still under development)\n",
+    "\n",
+    "## Installation\n",
+    "\n",
+    "If you want to use icon-vis on your own machine, please follow the [installation instruction](https://github.com/C2SM/icon-vis#environment-setup) in the README. For this course, an environment is already prepared for you to save time."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6ddf5f4a",
+   "metadata": {},
+   "source": [
+    "## First we will download some example data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5fe79a4d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get example datasets for testing (won't do anything if data is already downloaded)\n",
+    "import iconarray as iconvis # import self-written modules from iconarray\n",
+    "iconvis.get_example_data()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "36ce8c7c",
+   "metadata": {},
+   "source": [
+    "## Exercise\n",
+    "- Checkout the [example gallery](https://github.com/C2SM/icon-vis#example-plots) and select a plot you would like to produce and follow the link to the folder containing the example. Note that there is no notebook for the transect plot. The edge variable plot and the vector plot only work after sourcing setup-conda-env.sh, which has currently not been done for this environment.\n",
+    "- Every folder contains a jupyter notebook. Give it a try and play around with it."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7a15aad2",
+   "metadata": {},
+   "source": [
+    "## Exercise\n",
+    "For some of the plots, there is also a Python script given, namely for the difference map, the map plot, the timeseries and the vertical profile.\n",
```

Comment:
Above you mention jupyter notebooks but here it's scripts?

Reply:
For every plot there is a jupyter notebook and for some, there is also a Python script as the sentence says `For some of the plots, there is also a Python script given, namely for the difference map, the map plot, the timeseries and the vertical profile.`